src/compose/types/service.ts

```diff
@@ -184,7 +184,7 @@ export interface ConfigMap {
 // is typescript
 export interface DeviceMetadata {
 	imageInfo?: Dockerode.ImageInspectInfo;
-	uuid: string;
+	uuid: Nullable<string>;
```

Comment:
Is this better as
```suggestion
	uuid?: string | null;
```
?